.ci.yaml

```diff
@@ -1683,7 +1683,6 @@ targets:
       dependencies: >-
         [
           {"dependency": "android_sdk"},
```

Comment:
FYI: This trailing comma needs to be removed become of the beauty of JSON :-)

Reply:
ahh yes, ofc!